src/help/guides/extension-development/intro-to-sources.md

```diff
@@ -1,12 +1,23 @@
 ### Introduction: A Manga Reader Without Servers
-When developing a Manga Reader, first and foremost, you need to gather your content from somewhere. There are many services out there, such as WebToons or MangaDex which run their own content providing system which their applications can attach to without much issues.
 
-If you do not hold this kind of infrastructure, as Paperback does not, the app needs to be designed in a way where it can dynamically read **other** content providers.
+When developing a Manga Reader, first and foremost, you need to gather your content from somewhere. There are many
+services out there, such as WebToons or MangaDex which run their own content providing system which their applications
+can attach to without many issues.
```

Comment:
Do we want to cut this text in multiple lines? It is something not done on the rest of the website

Reply:
I planned on doing so for the rest of the website, as markdown only considers a double newline a new paragraph. It does help editing files if you don't have wrapping on.

Reply:
Keeping it consistent is a good call. I see that you **are** keeping it consistent with the multiple lines through this PR. I'm a little unsure how I feel about it, much preferring the one-line style letting your IDE do the wrapping, but this isn't a dealbreaker I guess.